src/cpp/common/alarm.cc

```diff
@@ -31,21 +31,14 @@
  */
 
 #include <grpc++/alarm.h>
-#include <grpc++/completion_queue.h>
 #include <grpc++/impl/grpc_library.h>
-#include <grpc/grpc.h>
 
 namespace grpc {
 
-static internal::GrpcLibraryInitializer g_gli_initializer;
-Alarm::Alarm(CompletionQueue* cq, gpr_timespec deadline, void* tag)
-    : tag_(tag),
-      alarm_(grpc_alarm_create(cq->cq(), deadline, static_cast<void*>(&tag_))) {
-  g_gli_initializer.summon();
+Alarm::~Alarm() {
+  grpc_alarm_destroy(alarm_);
 }
 
-Alarm::~Alarm() { grpc_alarm_destroy(alarm_); }
-
 void Alarm::Cancel() { grpc_alarm_cancel(alarm_); }
 
```

Comment:
Given the trivialness of ~Alarm and Cancel and the removal of the g_gli_initializer, is there any continuing need for alarm.cc . Can't this all just go in the .h? Deleted file is best file.